Jenkinsfile_CNP

```diff
@@ -10,10 +10,14 @@ properties([
                 choice(name: 'PRODUCT_NAME', choices: 'ccd\nprobate\nsweeper\nccdcmc\nccddivorce\nccdprobate\nccdsscs\nccdethos', description: 'product name like probate, ccd, div.'),
                 booleanParam(name: 'BUILD_LOGSTASH_IMAGE', defaultValue: false, description: 'set to true to build a new Logstash image'),
                 booleanParam(name: 'DEPLOY_LOGSTASH', defaultValue: false, description: 'set to true to deploy Logstash'),
+                stringParam(name: 'START_CASE_ID', defaultValue: false, description: 'the DB case ID from which the Logstash Indexer needs to start indexing from'),
+                stringParam(name: 'END_CASE_ID', defaultValue: false, description: 'the DB case ID up to which the Logstash Indexer needs to index to'),
         ])
 ])
 
 productName = params.PRODUCT_NAME
+startCaseId = params.START_CASE_ID
```

Comment:
ah need to find a way to change the LS instance ID otherwise deploy will fail.
WIP